news/_posts/2020-06-03-a-definition-of-the-good-first-issue-label.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "A definition of the ‘good first issue’ label"
```

Comment:
```suggestion
title:  "A definition of the 'good first issue' label"
```

You should change the Word character and use `'` instead (everywhere in the content)